lrauv_system_tests/test/vehicle/test_battery_full_charge.cc

```diff
@@ -45,25 +45,26 @@ TEST(BatteryTest, TestDischargeFullCharged)
 
   fixture.Step(1000u);
 
-  auto batteryMsgs = batterySubscription.ReadMessages();
-  int n = batteryMsgs.size() - 1;
-  double initialCharge = batteryMsgs[0].charge();
-  
+  EXPECT_GT(batterySubscription.MessageHistorySize(), 5);
```

Comment:
@adityapande-1995 meta: the good thing about using something like `ASSERT_TRUE(batterySubscription.WaitForMessages(5, 10s))` instead is that you don't need to rely on messages to have been delivered in the amount of time it took to step the simulation forward.

Either way, this should probably be an `ASSERT`. Same elsewhere.

Reply:
Done here : https://github.com/osrf/lrauv/pull/207/commits/6f241f440de4b3a1c51ef7f67880c6bc014d581f